source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs

```diff
@@ -182,10 +182,10 @@ private void UpdateOutputs()
                             CoordinateMGRS.TryParse(coord, out mgrs))
                         {
                             output.OutputCoordinate = mgrs.ToString(output.Format, new CoordinateMGRSFormatter());
-                            props.Add("GZD", mgrs.GZD);
-                            props.Add("Grid Sq", mgrs.GS);
-                            props.Add("Easting", mgrs.Easting.ToString("00000"));
-                            props.Add("Northing", mgrs.Northing.ToString("00000"));
+                            props.Add(Properties.Resources.StringGZD, mgrs.GZD);
+                            props.Add(Properties.Resources.StringGridSq, mgrs.GS);
+                            props.Add(Properties.Resources.StringEasting, mgrs.Easting.ToString(Properties.Resources.StringZeros));
+                            props.Add(Properties.Resources.StringNorthing, mgrs.Northing.ToString(Properties.Resources.StringZeros));
```

Comment:
undo change, nothing has changed, are you pushing your commits?  do not edit .ToString()